Dockerfile

```diff
@@ -44,12 +76,23 @@ COPY pyproject.toml .
 COPY README.md .
 COPY LICENSE .
 
+# Copy libdispatch from the build stage
+COPY --from=libdispatch-build /usr/lib/x86_64-linux-gnu/libdispatch.so* /usr/lib/x86_64-linux-gnu/
+
+# Copy and verify the strip and ld binaries, then make them executable
+COPY scripts/strip/dist/strip scripts/strip/dist/ld /app/scripts/strip/dist/
+RUN echo "4cd01dd28294a3ebeff031d6ba947aee1c2dd9c402f504f9866eec302466b11d  /app/scripts/strip/dist/strip" | sha256sum -c - && \
```

Comment:
Added checksum verification for the binaries, I think our `deps` does this too once I migrate over to using that.